charts/ark-pentaho-ce/files/start-pentaho.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+# *******************************************************************************************
+# This program is free software; you can redistribute it and/or modify it under the
+# terms of the GNU General Public License, version 2 as published by the Free Software
+# Foundation.
+#
+# You should have received a copy of the GNU General Public License along with this
+# program; if not, you can obtain a copy at http://www.gnu.org/licenses/gpl-2.0.html
+# or from the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+#
+# Copyright 2011 - 2018 Hitachi Vantara.  All rights reserved.
+# *******************************************************************************************
+
+### ====================================================================== ###
+##                                                                          ##
+##  Hitachi Vantara Start Script                                                    ##
+##                                                                          ##
+### ====================================================================== ###
+
+DIR_REL=`dirname $0`
+cd $DIR_REL
+DIR=`pwd`
+#cd -
+
+. "$DIR/set-pentaho-env.sh"
+
+setPentahoEnv "$DIR/jre"
+
+### =========================================================== ###
+## Set a variable for DI_HOME (to be used as a system property)  ##
+## The plugin loading system for kettle needs this set to know   ##
+## where to load the plugins from                                ##
+### =========================================================== ###
+DI_HOME="$DIR"/pentaho-solutions/system/kettle
+
+errCode=0
+if [ -f "$DIR/promptuser.sh" ]; then
+  sh "$DIR/promptuser.sh"
+  errCode="$?"
+  rm "$DIR/promptuser.sh"
+fi
+if [ "$errCode" = 0 ]; then
+  cd "$DIR/tomcat/bin"
+  CATALINA_OPTS="-Xms2048m -Xmx6144m -XX:MaxPermSize=256m -Dsun.rmi.dgc.client.gcInterval=3600000 -Dsun.rmi.dgc.server.gcInterval=3600000 -Dfile.encoding=utf8 -DDI_HOME=\"$DI_HOME\""
+  export CATALINA_OPTS
+  JAVA_HOME=$_PENTAHO_JAVA_HOME
+  sh catalina.sh run
```

Comment:
The last line of the startup script should be `exec ...`, not just to fork a shell. See the end of [this](https://arkcase.atlassian.net/wiki/spaces/AANTA/pages/1398046744/Best+practices+for+building+container+images#How-to-pass-parameters-to-the-containerized-app) section.

Reply:
Tried to run with `exec`  but pod is going to CrashBackLoop status. I think we cannot use it with Pentaho.

Reply:
OK, fair enough